bundle/src/main/java/com/adobe/acs/commons/wcm/impl/AemEnvironmentIndicatorFilter.java

```diff
@@ -385,4 +425,43 @@ String getTitlePrefix() {
     String[] getExcludedWCMModes() {
         return excludedWCMModes;
     }
+    
+    
+    private class InnerEnvironmentIndicatorFilter implements Filter {
+
+    	String[] excludedWcmModes;
+
+    	public InnerEnvironmentIndicatorFilter(String[] excludedWcmModes) {
+			this.excludedWcmModes = excludedWcmModes;
+		}
+
+		@Override
+		public void init(FilterConfig filterConfig) throws ServletException {
+			// ignore
+			
+		}
+
+		@Override
+		public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain)
+				throws IOException, ServletException {
+
+		    SlingHttpServletRequest req = (SlingHttpServletRequest) request;
```

Comment:
Unused variable can be removed

Reply:
It's still used within the next block to write log statements to the RequestProgressTracker